geomesa-raster/src/main/scala/org/locationtech/geomesa/raster/package.scala

```diff
@@ -19,7 +19,13 @@ package org.locationtech.geomesa
 import org.calrissian.mango.types.LexiTypeEncoders
 
 package object raster {
-  def lexiEncodeDoubleToString(number: Double): String = LexiTypeEncoders.LEXI_TYPES.encode(number)
+  def lexiEncodeDoubleToString(number: Double): String = {
```

Comment:
I think it would be wise to add some comments here as to what is being done. WHY it is being done can either go here or where the functions are called.